extensions/wikia/AdEngine/js/tracking/adInfoTracker.js

```diff
@@ -111,6 +111,7 @@ define('ext.wikia.adEngine.tracking.adInfoTracker',  [
 			'btl': (billTheLizard && billTheLizard.serialize()) || '',
 			'page_width': win.document.body.scrollWidth || '',
 			'page_layout': pageLayout.getSerializedData(slotName) || '',
+			'document_visibility': bridge.geo.getDocumentVisibilityStatus(),
```

Comment:
geo? How does it work?

Reply:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/src/ad-engine.bridge.js#L274
Got it 🤦‍♂️